### PR TITLE
Add details on remote sessions to the docs [skip ci]

### LINKED ISSUE #970: Add documentation on the session param for remote connections
Right now there's no mention in the docs about using a sessioned connection. When going through the  configuredgraphfactory examples I was unaware that commands all needed to be on a single line and it created some frustration, other new users might run into a similar situation.  I think the appropriate spot would be around 7.1.1.1.

### PR DESCRIPTION
**Pull request by chupman**
fixes #970 

Signed-off-by: Chris Hupman <chupman@us.ibm.com>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [X] Is there an issue associated with this PR? Is it referenced in the commit message?
- [X] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [X] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [X] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [X] Have you ensured that format looks appropriate for the output in which it is rendered?
- [X] If this PR is a documentation-only change, have you added a `[skip ci]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by chupman** [comment]
@mbrukman base updated to 0.2

File: docs/basics.adoc [hunk 1]
```diff
@@ -530,7 +530,26 @@ gremlin> :> g.V().values('name')
 ==>stephen
 ----
 
-The `:remote` command tells the console to configure a remote connection to Gremlin Server using the `conf/remote.yaml` file to connect.  That file points to a Gremlin Server instance running on `localhost`.  The `:>` is the "submit" command which sends the Gremlin on that line to the currently active remote.
+The `:remote` command tells the console to configure a remote connection to Gremlin Server using the `conf/remote.yaml` file to connect.  That file points to a Gremlin Server instance running on `localhost`.  The `:>` is the "submit" command which sends the Gremlin on that line to the currently active remote. By default the console is sessionless. If you wish to use variables with a remote connection you can either run the entire command on a single line or append session to the remote connect command.
```
Comment (pluradj) [inline docs/basics.adoc]: By default, the console is sessionless, meaning that each line sent in the console is interpreted as a single request. Multiple statements can be sent on a single line using a semicolon as the delimiter.

Alternately, you can establish a console with a session by specifying `session` when creating the connection. A console session allows you to reuse variables across several lines of input.

Also include references back to the TinkerPop docs on this topic:
- https://tinkerpop.apache.org/docs/current/reference/#console-sessions
- https://tinkerpop.apache.org/docs/current/reference/#sessions

File: docs/basics.adoc [hunk 2]
```diff
@@ -530,7 +530,26 @@ gremlin> :> g.V().values('name')
...
+
+----
+gremlin> :remote connect tinkerpop.server conf/remote.yaml
+==>Configured localhost/127.0.0.1:8182
+gremlin> :remote console
+==>All scripts will now be sent to Gremlin Server - [localhost/127.0.0.1:8182] - type ':remote console' to return to local mode
+gremlin> foo = "bar"
```
Comment (pluradj) [inline docs/basics.adoc]: Doing a simple operation like this is ok for showing that the server is up and responding to commands. The TinkerPop docs does a math example.

It'd be good to show a graph operation using `g` also.

**Comment by pluradj** [comment]
needs to be merged to `master`

**Pull request merged**